Jenkinsfile

```diff
@@ -9,44 +9,56 @@ pipeline {
           dir 'test'
         }
       }
-
       steps {
         sh 'npm install -f'
         sh 'npm run lint'
         sh 'npm run test'
       }
-
       post {
         always {
           deleteDir() /* clean up our workspace */
         }
       }
     }
-
-    stage('Deliver Docker images') {
+    stage('Release latest image on Docker hub') {
       when { branch 'main' }
       agent {
         docker {
           image 'docker:19.03.12-dind'
           args '-e DOCKER_HOST=$DOCKER_HOST'
         }
       }
-
       steps {
         script {
           def dockerImage = docker.build 'linagora/esn-frontend-inbox'
           docker.withRegistry('', 'dockerHub') {
-            dockerImage.push('branch-main')
+            dockerImage.push('latest')
```

Comment:
you will change all others Jenkinsfile to `latest` ?

Reply:
yes exactly